NewsApp/NewsApp/Controller/ProfileController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ProfileController.swift
+//  NewsApp
+//
+//  Created by Josselin DUBOC on 26/03/2020.
+//  Copyright © 2020 Josselin DUBOC. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ProfileController: UITableViewController {
+    @IBOutlet private weak var button: UIButton!
```

Comment:
Idem que plus haut essaie de préciser à quoi sert le bouton ;)

Reply:
fix in "fix PR"